### DISCUSSION TITLE
Visual Studio error when trying to target a different version of the file

### DISCUSSION THREAD

**Discussion by StevenRasmussen** [VsixCommunity/Community.VisualStudio.Toolkit#44] [General]
Is anyone else experiencing an issue with this repo as described here?
[https://developercommunity.visualstudio.com/t/systemargumentoutofrangeexception-specified-argume-2/1427702](https://developercommunity.visualstudio.com/t/systemargumentoutofrangeexception-specified-argume-2/1427702)

May [latest comment](https://developercommunity.visualstudio.com/t/systemargumentoutofrangeexception-specified-argume-2/1427702#T-N1430605) references this repo and demonstrates the issue.  I don't think it's necessarily and issue with this repo specifically but wondering if others are experiencing this issue as well.

**Comment by madskristensen** [comment]
Does this happen when using the 14.0 toolkit only or newer versions as well?

**Reply by StevenRasmussen** [reply]
Basically when I try to use that dropdown and switch to any version (including the version that is already selected) that error happens.

**Reply by madskristensen** [reply]
I've not seen the error before, but I have noticed that I can't change the target version in the dropdown. It reverts back to what it was previuosly after hanging for a few seconds. Perhaps that's the error to report through Developer Community instead

**Reply by StevenRasmussen** [reply]
Right... that's the behavior that I'm noticing along with the error... that's strange that you don't see the error though.  At least it appears I'm not the only one affected though so that's good to confirm. Thanks!

**Reply by reduckted** [reply]
I haven't tried switching between targets with the most recent Visual Studio update, but yes, I did encounter this a few weeks ago. It's basically the same thing that Mads mentioned - it pauses and then reverts to the previous target.

I'll see if I can replicate it again and add a comment to that issue you linked to.

**Reply by reduckted** [reply]
Yep, just tried again and it's still occurring with 16.10.

@StevenRasmussen as a workaround, I found that if, for example, you're trying to select `Community.VisualStudio.Toolkit.14.0` as the target from the dropdown, then if you unload the 15.0, 16.0 and 17.0 projects (and possibly close and reopen the solution), then `Community.VisualStudio.Toolkit.14.0` will become the target because it's the only item left in the dropdown.

**Reply by StevenRasmussen** [reply]
@reduckted - Nice!  Hadn't thought of doing that. Thanks!
